app/views/responsible/academics/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<% provide(:page_title, t('breadcrumbs.academics.index')) %>
+<% provide(:head_title, t('breadcrumbs.academics.index')) %>
+
+
+<div class="row">
+  <div class="col-8">
+    <search url="<%= responsible_academics_search_path %>" />
+  </div>
+  <div class="col-4">
+    <%= link_to(new_responsible_academic_path, class: 'btn btn-outline-primary d-block') do %>
+      <i class="fe fe-plus"></i>
+      <%= t('breadcrumbs.academics.new') %>
+    <% end %>
+  </div>
+</div>
+
+<div class="table-striped mt-5 table-responsive w-100">
+  <table class="table card-table table-vcenter text-nowrap">
+    <thead>
+      <tr>
+        <th><%= Academic.human_attribute_name('name') %></th>
+        <th><%= Academic.human_attribute_name('email') %></th>
+        <th><%= Academic.human_attribute_name('ra') %></th>
+        <th><%= Academic.human_attribute_name('created_at') %></th>
+        <th colspan="3"></th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <%= render @academics %>
+    </tbody>
+  </table>
+</div>
+
+<%= paginate @academics %>
```

Comment:
![image](https://user-images.githubusercontent.com/22449115/53341789-3e960580-38eb-11e9-9633-236b8e02137d.png)

The page number doesn't change properly.